### PR TITLE
Switch to Github Actions

### PR DESCRIPTION
**Pull request by ekohl**
This drops testing of various Python versions on Ubuntu and adds testing to on CentOS 8. Since we only have integration tests, this is not a problem.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+
+  centos:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        centos: ["7", "8"]
+    container:
```
Comment (ekohl) [inline .github/workflows/main.yml:40]: @johnpmitsch you might find this interesting because it allows you to run on CentOS instead of Ubuntu.

Comment (johnpmitsch) [reply .github/workflows/main.yml:40]: interesting! So is it a container running on ubuntu?

Comment (ekohl) [reply .github/workflows/main.yml:40]: Yes

**Comment by ehelms** [comment]
Should this have kicked off a test run?

**Comment by johnpmitsch** [comment]
> Should this have kicked off a test run?

It appears if you don't have any github workflow already it won't kick them off in the PR stage. But if they are some present in master, they will kick off a new one in a PR.

This is based on adding a workflow in a PR to Katello and seeing it run, presumably because there is a workflow already merged. https://github.com/Katello/katello/pull/8722

**Comment by ekohl** [comment]
Correct. See https://github.com/ekohl/katello-certs-tools/actions/runs/109593497 for the result of this.

**Comment by ekohl** [comment]
Run on my fork: https://github.com/ekohl/katello-certs-tools/actions/runs/124041783
After merging this will run as well for new PRs. @evgeni mentioned that you can also push an empty file to master and then it'll run here as well, but that's a bit overkill to me.

**Pull request merged**